Source/ASButtonNode.mm

```diff
@@ -131,6 +131,55 @@ - (void)setDisplaysAsynchronously:(BOOL)displaysAsynchronously
   [self.titleNode setDisplaysAsynchronously:displaysAsynchronously];
 }
 
+
+- (NSAttributedString *)__updateTitle:(NSAttributedString *)title withForegroundColor:(UIColor *)newColor
+{
+  if (title) {
+    NSMutableAttributedString *mutString = [[NSMutableAttributedString alloc] initWithAttributedString:title];
+    NSRange limit = NSMakeRange(0, _normalAttributedTitle.length);
```

Comment:
should this be `title`, not `_normalAttributedTitle`? It looks like several different titles are being passed in here in `updateTitleForegroundColor`

Reply:
Great catch!